osu.Game/OsuGame.cs

```diff
@@ -315,8 +315,12 @@ private void load()
         /// The user should have already requested this interactively.
         /// </summary>
         /// <param name="beatmap">The beatmap to select.</param>
-        public void PresentBeatmap(BeatmapSetInfo beatmap)
+        /// <param name="findPredicate">Predicate used to find a difficulty to select</param>
```

Comment:
I'd describe this properly and remove the inline comment:
```suggestion
        /// <param name="findPredicate">
        /// Optional predicate used to try and find a difficulty to select.
        /// If omitted, this will try to present the first beatmap from the current ruleset.
        /// In case of failure the first difficulty of the set will be presented, ignoring the predicate.
        /// </param>
```
Also I'd maybe rename to something like `difficultyCriteria`.